client/app/app.js

```diff
@@ -211,4 +205,17 @@ angular
   .run(($translate) => {
     moment.locale(_.first($translate.use().split('_')));
   })
+  .run((constants, atInternet, OvhApiMe) => {
+    const level2 = constants.target === 'US' ? '57' : '10';
```

Comment:
Theses values shouldn't be moved into a constant file? In order to facilitate any re-use.